README.md

```diff
@@ -105,7 +105,7 @@ Pre-requisites:
 - The commit which will be submitted must be a valid NVDA `*.nvda-addon` file if zipped and renamed.
 
 Process:
-1. Addon author creates a new Pull Request (PR) on the 'NVDA-Addon-submission' repository. In most cases this could be done with the web editor:
+1. Addon author creates a new Pull Request (PR) on the 'NVDA-Addon-submission' repository. This could be done with the web editor, though screen reader users may be more comfortable making it locally with their chosen editor or using a provided template:
```

Comment:
Perhaps this should be expanded to include plans for submitting via an issue? If we do that, then I would remove the instructions for how to edit the file, this becomes the 'advanced' workflow for people familiar with the concepts.

Reply:
I think it would be easier to send issues and remove instructions to edit the file.